install/resources/mybb_theme.xml

```diff
@@ -822,6 +822,14 @@ blockquote cite span.highlight {
 	font-size: 13px;
 }
 
+.mycode_me {
```

Comment:
Unfortunately adding new CSS classes requires updating all existing themes. We should avoid that in a minor update to avoid support threads (although most users probably won't note it in this case). IMHO it either should have done in 1.8.0 or be deferred to 2.0 unless you have another solution.